dinky-metadata/dinky-metadata-kyuubi/src/main/java/org/dinky/metadata/constant/KyuubiConstant.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.dinky.metadata.constant;
+
+public interface KyuubiConstant {
+
+    /** 查询所有database */
+    String QUERY_ALL_DATABASE = " show databases";
```

Comment:
I didn't have a deep look at where it is used, but generally, you should rely on standard JDBC API instead of SQL dialect to retrieve such metadata.